symengine/lib/symengine_wrapper.pyx

```diff
@@ -2714,6 +2732,36 @@ def module_cleanup():
 import atexit
 atexit.register(module_cleanup)
 
+def add_expand(a):
+    cdef Basic _a = sympify(a)
+    require(_a, Add)
+    cdef RCP[const symengine.Add] a1 = symengine.rcp_static_cast_Add(_a.thisptr)
+    return c2py(symengine.add_expand(a1))
+
+expand_add = add_expand
+
+def mul_expand(a):
+    cdef Basic _a = sympify(a)
+    require(_a, Mul)
+    cdef RCP[const symengine.Mul] a1 = symengine.rcp_static_cast_Mul(_a.thisptr)
+    return c2py(symengine.mul_expand(a1))
+
+expand_mul = mul_expand
+
+def pow_expand(a):
+    cdef Basic _a = sympify(a)
+    require(_a, Pow)
+    cdef RCP[const symengine.Pow] a1 = symengine.rcp_static_cast_Pow(_a.thisptr)
+    return c2py(symengine.pow_expand(a1))
+
+expand_pow = pow_expand
```

Comment:
@isuruf Can you tell me why the above code is failing? More specifically what can be done to prevent this:
```
symengine_wrapper.obj : error LNK2019: unresolved external symbol "class SymEngine::RCP<class SymEngine::Basic const > __cdecl SymEngine::pow_expand(class SymEngine::RCP<class SymEngine::Pow const > const &)" (?pow_expand@SymEngine@@YA?AV?$RCP@$$CBVBasic@SymEngine@@@1@ABV?$RCP@$$CBVPow@SymEngine@@@1@@Z) referenced in function "struct _object * __cdecl __pyx_pf_9symengine_3lib_17symengine_wrapper_36pow_expand(struct _object *,struct _object *)" (?__pyx_pf_9symengine_3lib_17symengine_wrapper_36pow_expand@@YAPAU_object@@PAU1@0@Z) [C:\projects\symengine-py\build\lib.win32-2.7\symengine\lib\symengine_wrapper.vcxproj]
```